zha/application/platforms/binary_sensor/__init__.py

```diff
@@ -197,12 +200,17 @@ class Motion(Opening):
 
     _attr_device_class: BinarySensorDeviceClass = BinarySensorDeviceClass.MOTION
 
+    # Many devices support both "motion" and "occupancy", with "motion" being the more
+    # useful entity to have
+    _attr_primary_weight = 3
```

Comment:
True, but this `Motion` class is the `OnOff` one, only for IKEA sensors and older Hue sensors.
IKEA motion sensors only have an `OnOff` entity, no occupancy or IAS one.

Hue sensors have the `OnOff` entity (doesn't actually work on SML001, SML002 by default IIRC, only on SML003 and SML004 that aren't even matched here), but also the occupancy entity.
The occupancy entity is actually the motion one, so it times out after 10 seconds.
This "motion"/OnOff one times out after a couple of minutes.

I'd argue this should be `_attr_primary_weight = 1`.